tensorrt_llm/llmapi/llm.py

```diff
@@ -303,6 +303,7 @@ def generate_async(
         lora_request: Optional[LoRARequest] = None,
         prompt_adapter_request: Optional[PromptAdapterRequest] = None,
         streaming: bool = False,
+        attention_dp_rank: Optional[int] = None,
```

Comment:
We have this parameter inside a class so that we don't need to break the API again in the future if we add more scheduling related parameters. Maybe something like `schedulng_params`. In the future, we could include request priority (which we have in the C++ executor API) for example. @Superjomn what do you think?